host-bin/enter-chroot

```diff
@@ -355,6 +390,10 @@ tmpfsmount /var/run/lock 'noexec,nosuid,nodev,size=5120k'
 bindmount /var/run/dbus /var/host/dbus
```

Comment:
`2>/dev/null || true`. We probably won't bind mount /var/run/dbus in verified mode.